scripts/win/install-protobuf.bat

```diff
@@ -0,0 +1,6 @@
+ECHO Fetching protoc...
+powershell.exe -nologo -noprofile -command "& { [Net.ServicePointManager]::SecurityProtocol = 'tls12, tls11, tls'; (new-object System.Net.WebClient).DownloadFile('https://github.com/protocolbuffers/protobuf/releases/download/v3.6.1/protoc-3.6.1-win32.zip','%TEMP%\protoc-3.6.1-win32.zip'); }"
+ECHO Extracting protoc...
+powershell.exe -nologo -noprofile -command "& { Add-Type -A 'System.IO.Compression.FileSystem'; [IO.Compression.ZipFile]::ExtractToDirectory('%TEMP%\protoc-3.6.1-win32.zip', '%USERPROFILE%\protoc-3.6.1'); }"
```

Comment:
@antonlerner The last version of Windows not to include it was Vista, which is now over 12 years old with less than 1% market share and stopped receiving critical security updates years ago. Do you think it's reasonable to leave it out from prerequisites to avoid cluttering the README with unnecessary information?